app/keepers/keepers.go

```diff
@@ -677,23 +692,34 @@ func (ak *AppKeepers) InitKeepers(
 	wasmStackIBCHandler := wasm.NewIBCHandler(ak.WasmKeeper, ak.IBCKeeper.ChannelKeeper, ak.IBCFeeKeeper)
 	wasmStack = ibcfee.NewIBCMiddleware(wasmStackIBCHandler, ak.IBCFeeKeeper)
 
+	// Create Transfer Stack (from bottom to top of stack)
+	// - core IBC
+	// - ratelimit
+	// - fee
+	// - PFM (Packet Forwarding Middleware)
+	// - callbacks
+	// - transfer
+
 	// Create Transfer Stack
 	// SendPacket Path:
-	// SendPacket -> Transfer -> Callbacks -> PFM -> Fee -> IBC core (ICS4Wrapper)
+	// SendPacket -> Transfer -> Callbacks -> PFM -> Fee -> RateLimit -> IBC core (ICS4Wrapper)
```

Comment:
I you look how atom wires up middlewares: https://github.com/cosmos/gaia/blob/release/v23.x/app/keepers/keepers.go#L517,
in the send path, they have PFM then Ratetlimiter and they pass rate limiter to pfm keeper as a ics4wrapper:
https://github.com/cosmos/gaia/blob/release/v23.x/app/keepers/keepers.go#L458,
thus to achieve wiring described here i.e `Fee -> RateLimit -> IBC core ` , imo we should pass rate limiter to `FeeKeeper` as  ics4wrapper i.e have:
```
	ak.IBCFeeKeeper = ibcfeekeeper.NewKeeper(
		appCodec, keys[ibcfeetypes.StoreKey],
		ak.RatelimitKeeper, // may be replaced with IBC middleware
		ak.IBCKeeper.ChannelKeeper,
		ak.IBCKeeper.PortKeeper, ak.AccountKeeper, ak.BankKeeper,
	)
```